app.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+require "sinatra/base"
+require "./conda"
```

Comment:
I think `require_relative "conda"` might be more idiomatic, or even better, adjusting your load path so you can just `require "conda"`